### DISCUSSION TITLE
Manage Settings in Cookie consent banner

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#4355] [Q&A] [answered]
Hello,
I believe the purpose of the **Manage Settings** button on the Cookie consent banner is to let users configure the privacy settings. However, when i click it, nothing happens. I am expecting that a dialog box opens where the user can configure the settings. Am i missing anything?

Sample site - https://syeda-git.github.io/mkdocs-sandbox/site/

Syed

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. Please share your configuration.

**Reply by syeda-git** [reply]
@squidfunk Please check here
https://github.com/syeda-git/mkdocs-sandbox

**Reply by squidfunk** [reply]
[Your configuration is wrong again](https://github.com/syeda-git/mkdocs-sandbox/blob/2a2c96e53240e5a47bce6817c0e70e71035fcc1c/mkdocs.yml#L51-L54). [Please stick to what we write in the documentation](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#__tabbed_1_1).

**Reply by syeda-git** [reply]
Thanks. After fixing the configuration, i can see the GA checkbox when the Manage Settings button is clicked. By default, the checkbox is selected. Does this mean that if you don't want GA to track, you should deselect the checkbox?

Second question - How can i use my companies' Cookie consent banner? Check this page for the banner.
https://help.hcltechsw.com/

**Reply by squidfunk** [reply]
1. Yes, but you can also change it to default to unchecked. [Please see our documentation for reference](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/#__tabbed_1_2).
2. You can customize the appearance by [extending the theme](https://squidfunk.github.io/mkdocs-material/customization/).

**Comment by syeda-git** [comment]
Hi Martin,

My company uses a vendor for managing Privacy, cookie consent being one of
the aspects of Privacy. To use their cookie banner on our site's pages, we
have to add the following code within the head tag. When I add the code to
any HTML file, I can see the banner. See the attached sample HTML file.

I added the same code to the extrahead block in main.html so it shows up
within the head tags of the HTML files. The code is in the head tags of the
HTML files. However, the cookie banner doesn't show up when you open the
page in a browser.

Would you know why the code isn't working

Sample MkDocs site -
https://syeda-git.github.io/mkdocs-sandbox/site/about/about.html
Source package - https://github.com/syeda-git/mkdocs-sandbox/tree/main

Check https://help.hcltechsw.com for an example of how the banner and
buttons function.

<script type="text/javascript" src="
https://cdn.cookielaw.org/consent/99b8d579-c52a-43c9-ab7b-a35e60de7103/OtAutoBlock.js
"></script>
<script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="
text/javascript" charset="UTF-8" data-domain-script="
99b8d579-c52a-43c9-ab7b-a35e60de7103"></script>
<script type="text/javascript">
function OptanonWrapper() { }
</script>


-- 
Regards
Syed Attaullah.

**Reply by squidfunk** [reply]
> I added the same code to the extrahead block in main.html so it shows up
within the head tags of the HTML files. The code is in the head tags of the
HTML files.

Thanks for asking. Please understand, that we can only offer support for the native cookie consent. In general, when you're using customizations, you're on your own. It might be a good idea to ask on a general web development board like StackOverflow, since the code shows up in the output, so there's no error with Material for MkDocs.
